### PR TITLE
Add nr.* libs

### PR DESCRIPTION
**Pull request by NiklasRosenstein**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by NiklasRosenstein** [comment]
Hey @conda-forge/help-python, is there a way to have these recipes updated automatically when there is a new release on PyPI? Or how does it work with getting new versions into conda-forge? Thanks!

File: recipes/nr.bundler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f2217a0f1f2041ae3406d36a984ef14cc848aec6277a2b654235b7c40ea0cf5e"
+
+build:
+  number: 0
```
Comment (bgruening) [inline recipes/nr.bundler/meta.yaml]: This can be a `noarch: python` package isn't it?

File: recipes/nr.bundler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - python
+    - setuptools
+  run:
+    - appdirs >=1.4.3
+    - distlib >=0.2.7
+    - nr.fs >=1.3.1
+    - nr.types >=2.0.0
+    - pefile >=2017.11.5
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/nr.bundler/meta.yaml]: is setuptools needed as runtime dependency?

File: recipes/nr.bundler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - appdirs >=1.4.3
+    - distlib >=0.2.7
+    - nr.fs >=1.3.1
+    - nr.types >=2.0.0
+    - pefile >=2017.11.5
+    - pip
```
Comment (bgruening) [inline recipes/nr.bundler/meta.yaml]: I think everything but pip and python can be removed from the host dependency

File: recipes/nr.bundler/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - nr.bundler.utils
+    - nr.bundler.vendor
+
+about:
+  home: "https://github.com/NiklasRosenstein/python-nr.bundler"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.txt"
+  summary: "Bundle the modules of a Python application. (WIP)"
+  doc_url: ""
```
Comment (bgruening) [inline recipes/nr.bundler/meta.yaml]: pleaes fill out these keys or remove.

File: recipes/nr.cli/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1bf47100b8ca21b3f184ee6f27c34adcfb7007612f8f8aada570891ffed59bca"
+
+build:
+  number: 0
```
Comment (bgruening) [inline recipes/nr.cli/meta.yaml]: noarch?

File: recipes/nr.cli/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  entry_points:
+    - nr = nr.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/nr.cli/meta.yaml]: remove setuptools?

File: recipes/nr.cli/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    - nr
+  commands:
+    - nr --help
+
+about:
+  home: "https://github.com/NiklasRosenstein/python-nr.cli"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: ""
+  summary: "This package provides the nr command-line tool which then allows you to use any comman registered to the \"nr.cli.commands\" entrypoint."
```
Comment (bgruening) [inline recipes/nr.cli/meta.yaml]: can you fix this summary pleaes?

File: recipes/nr.config/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: "6f9986e507a247996ff0a78b760f483f5222e8e89d2507295af15005d0906c18"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - nr.types >=2.1.0
```
Comment (bgruening) [inline recipes/nr.config/meta.yaml]: you can probably remove nr.types and six from host

File: recipes/nr.bundler/meta.yaml (hunk 2 above)
Comment (NiklasRosenstein) [reply recipes/nr.bundler/meta.yaml]: Yes, setuptools is used in this library.

File: recipes/nr.bundler/meta.yaml (hunk 3 above)
Comment (NiklasRosenstein) [reply recipes/nr.bundler/meta.yaml]: Correct. Can setuptools also be removed? The setup script actually imports it. I built meta.yaml with `conda skeleton` and it put all these deps in the `host` section

File: recipes/nr.cli/meta.yaml (hunk 6 above)
Comment (NiklasRosenstein) [reply recipes/nr.cli/meta.yaml]: Same here, can it actually be removed from the host section if the setup script imports `setuptools`?

File: recipes/nr.cli/meta.yaml (hunk 7 above)
Comment (NiklasRosenstein) [reply recipes/nr.cli/meta.yaml]: What's wrong about this summary? It used to result in a YAML syntax error from what `conda skeleton` spit out originally, where the double quotes hadn't been escaped. I added the backslashes to escape the double quotes.

File: recipes/nr.cli/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - nr
+  commands:
+    - nr --help
+
+about:
+  home: "https://github.com/NiklasRosenstein/python-nr.cli"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: ""
```
Comment (scopatz) [inline recipes/nr.cli/meta.yaml]: this needs a license file

Comment (NiklasRosenstein) [reply recipes/nr.cli/meta.yaml]: Done!

File: recipes/nr.config/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    - nr
+
+about:
+  home: "https://github.com/NiklasRosenstein/python-nr.config"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.txt"
+  summary: "This packages provides a class-oriented API for parsing YAMl/JSON configs."
+  doc_url: ""
+  dev_url: ""
```
Comment (scopatz) [inline recipes/nr.config/meta.yaml]: Please remove these empty entries

Comment (NiklasRosenstein) [reply recipes/nr.config/meta.yaml]: Done.

**Comment by scopatz** [comment]
This lGTM. Is it ready to go?

**Comment by NiklasRosenstein** [comment]
Yes, thanks! :-)

**Comment by NiklasRosenstein** [comment]
Hey @bgruening could you elaborate how that works? I actually just release a new version of `nr.types` and would like to make sure this new version propagates to conda-forge! 😃 Thanks

**Comment by NiklasRosenstein** [comment]
@scopatz Hey I just noticed I didn't use my correct GitHub username in the maintainer config. how should I update I it? By creating individual PRs in the feedstocks?

**Comment by NiklasRosenstein** [comment]
@scopatz @bgruening Who can I talk to to get ownership of the new feedstocks that have been created via this PR?

**Comment by scopatz** [comment]
> @scopatz Hey I just noticed I didn't use my correct GitHub username in the maintainer config. how should I update I it? By creating individual PRs in the feedstocks?

Ahh yeah, put in a PR to each of them fixing the name and then ping @conda-forge/core to merge them.

**Comment by NiklasRosenstein** [comment]
Thanks @scopatz, will do. On a side note, it seems I can't *really* tag  @conda-forge/core.

**Comment by scopatz** [comment]
You will be able to once you have your first feedstock.

**Pull request merged**